Procfile

```diff
@@ -1 +1 @@
-web: node server.js
+web: bash scripts/start-scalingo.sh
```

Comment:
Dans un Procfile, il me semble qu'on peut avoir un postdeploy https://doc.scalingo.com/platform/app/postdeploy-hook
Ce qui pourrait donner 
```
postdeploy: bash scripts/scalingo-police-noto.sh
web: node server.js
```

À toi de voir si c'est une bonne idée 😁

Reply:
Alors, déjà il faudrait le faire en `predeloy` plutôt, et en plus ils expliquent bien que c'est des [actions qui sont éxécutées dans un autre container ](https://doc.scalingo.com/platform/app/postdeploy-hook#what-you-should-not-do)

Reply:
On peut visiblement [déclarer](https://doc.scalingo.com/languages/nodejs/start#nodejs-build-hooks) un `scalingo-postbuild` dans notre `package.json` plutôt.